cypress/pages/workflow-executions-tab.ts

```diff
@@ -0,0 +1,51 @@
+import { BasePage } from "./base";
+import { WorkflowPage } from "./workflow";
+
+const workflowPage = new WorkflowPage();
+
+export class WorkflowExecutionsTab extends BasePage {
+	getters = {
+		executionsTabButton: () => cy.getByTestId('radio-button-executions'),
+		executionsSidebar: () => cy.getByTestId('executions-sidebar'),
+		autoRefreshCheckBox: () => cy.getByTestId('auto-refresh-checkbox'),
+		executionsList: () => cy.getByTestId('current-executions-list'),
+		executionListItems: () => this.getters.executionsList().find('div.execution-card'),
+		successfulExecutionListItems: () => cy.get('[data-test-execution-status="success"]'),
+		failedExecutionListItems: () => cy.get('[data-test-execution-status="error"]'),
+		executionCard: (executionId: string) => cy.getByTestId(`execution-details-${executionId}`),
+		executionPreviewDetails: () => cy.get('[data-test-id^="execution-preview-details-"]'),
+		executionPreviewDetailsById: (executionId: string) => cy.getByTestId(`execution-preview-details-${executionId}`),
+		executionPreviewTime: () => this.getters.executionPreviewDetails().find('[data-test-id="execution-time"]'),
+		executionPreviewStatus: () => this.getters.executionPreviewDetails().find('[data-test-id="execution-preview-label"]'),
+		executionPreviewId: () => this.getters.executionPreviewDetails().find('[data-test-id="execution-preview-id"]'),
+	};
+	actions = {
+		prepareExecutions: () => {
```

Comment:
this might be better as a utility for that test suite.. considering the wait in a loop here.. maybe we don't want to reuse this a lot.. also we could come up with a better name.. `createMockExecutions` or something